core/src/main/java/software/amazon/awssdk/core/RequestOverrideConfiguration.java

```diff
@@ -280,7 +280,5 @@ public B addApiName(Consumer<ApiName.Builder> apiNameConsumer) {
             addApiName(b.build());
             return (B) this;
         }
-
-        public abstract RequestOverrideConfiguration build();
```

Comment:
Why remove?

Reply:
this is redundant since `build()` is already defined in the interface.

Reply:
Does this scope down the return type at all?

Reply:
Nope, the interface is in the same class actually and they look exactly the same in this abstract class and interface. sonarlint gave me warning on this one.